aii-core/src/main/perl/Shellfe.pm

```diff
@@ -537,6 +538,10 @@ sub dhcp
 {
     my ($self, $node, $st, $cmd, $dhcpmgr) = @_;
 
+    # If the profile has a discovery plugin configured, then don't second-guess
+    # it - it may or may not be ISC DHCP
+    return if $st->{configuration}->elementExists (DISCOVERY);
```

Comment:
add a `$self->verbose("discovery plugin configured, no dhcp configuration")` or something like that